### ISSUE TITLE
Ludwig with Keras seed model

### ISSUE THREAD

**Issue by khayamgondal** [uber/ludwig#539]
Is there any way where I can provide a basic Keras model as a seed to Ludwig and Ludwig tries to fine-tune it?
For example,  I provide a model [input layer --> A conv layer --> Dense layer] and Ludwig takes this as a starting point and try to fine-tune it.

**Comment by w4nderlust** [comment]
@khayamgondal  unfortunately there's no way to do such a thing at the moment. f the model is simple and has not been trained on massive amoutns of data, meaning the training is not that expendive, then I would suggest to retrain the model.
